### PR TITLE
Add Security SIG proposal

### PR DESCRIPTION
**Pull request by kaywilliams**
Initial document

### CODE REVIEW

File: proposals/securitysig.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* Enable consumers of software to specify and implement policy over consumed software.
+* Enable administrators to inventory and audit software used within their organizations.
+* Enable detection and prevention of software tampering at runtime.
+* Provide mechanisms for breaches in the integrity of software to be communicated and remediated. 
+* Provide mechanisms for consumers to recover from compromised or untrusted software.
+
+## 5. A list of initial members, and a chair. There should be at least 3 different companies represented
+
+Initial members:
+* Microsoft
```
Comment (dlorenc) [inline proposals/securitysig.md]: Do you think you could include a few individuals from each company here as well?

File: proposals/securitysig.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* Google
+* CloudBees
+
+Chair: Kay Williams, Microsoft
```
Comment (dlorenc) [inline proposals/securitysig.md]: Email address/contact info would be helpful.

File: proposals/securitysig.md (hunk 1 above)
Comment (kaywilliams) [reply proposals/securitysig.md]: Done.

File: proposals/securitysig.md (hunk 2 above)
Comment (kaywilliams) [reply proposals/securitysig.md]: Added under the 'initial members' section.

**Comment by kohsuke** [comment]
When we merge this, let's not forget to move this into `sigs/`, which is where this should be.

**Comment by dlorenc** [comment]
I'll move and then merge.

**Comment by dlorenc** [comment]
Accepted in https://lists.cd.foundation/g/cdf-toc/message/219

**Pull request merged**